generators/angular/templates/src/main/webapp/app/entities/_entityFolder/list/_entityFile.component.html.ejs

```diff
@@ -181,22 +181,22 @@ _%>
                             </button>
     <%_ } _%>
 <%_ } _%>
-                            <button type="submit"
+                            <a
                                     [routerLink]="['/<%= entityPage %>', <%= entityInstance %>.<%= primaryKey.name %>, 'view']"
                                     class="btn btn-info btn-sm"
                                     data-cy="entityDetailsButton">
                                 <fa-icon icon="eye"></fa-icon>
                                 <span class="d-none d-md-inline" <%= jhiPrefix %>Translate="entity.action.view"><%- getWebappTranslation('entity.action.view') %></span>
-                            </button>
+                            </a>
 <%_ if (!readOnly) { _%>
 
-                            <button type="submit"
+                            <a
```

Comment:
The formatting looks a little off after this change. Can you try moving the `[routerLink]` line up with the `<a`? Running prettier might be a good idea too.